packages/common/react-components/src/components/Dialog/Dialog.tsx

```diff
@@ -30,13 +30,12 @@ export interface DialogProps {
   description?: ReactNode;
   children?: ReactNode;
   closeLabel?: string;
-  initiallyOpen?: boolean; // TODO(burdon): Change to just 'open'.
+  initiallyOpen?: boolean; // TODO(burdon): Change to just 'open' (i.e., controlled and non-controlled?)
```

Comment:
[Radix’s Dialog](https://www.radix-ui.com/docs/primitives/components/dialog)
> Can be controlled or uncontrolled.

Other aspects of r-c’s Dialog need to be refactored too, so I'll be sure to include this if not already addressed.